### PR TITLE
ci: Fix nits found by https://github.com/rhysd/actionlint

### PR DESCRIPTION
**Pull request by larseggert**

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -7,8 +7,6 @@ on:
     branches: ["main"]
     paths-ignore: ["*.md", "*.png", "*.svg", "LICENSE-*"]
   merge_group:
-    branches: ["main"]
-    paths-ignore: ["*.md", "*.png", "*.svg", "LICENSE-*"]
```
Comment (mxinden) [inline .github/workflows/check.yml:11]: Oh, I did not know. My bad.

I guess it is fine without, given that CI time isn't critical for GitHub Merge Queue runs?!

See also https://github.com/orgs/community/discussions/45899.

**Review by mxinden — APPROVED** [review]
Thank you for the catch!

File: .github/workflows/check.yml (hunk 1 above)
Comment (larseggert) [reply .github/workflows/check.yml:11]: I guess so. I was just getting annoyed at vscode complaining about the invalid components.

**Comment by mxinden** [comment]
CI failing in `Fetch NSS and NSPR` again.

**Review by martinthomson — APPROVED** [review]

**Pull request merged**